docs/source/ko/_toctree.yml

```diff
@@ -393,6 +393,8 @@
       - local: in_translation
         title: (번역중) Funnel Transformer
       - local: in_translation
+      - local: model_doc/gemma2
+        title: Gemma2
```

Comment:
There should be a `local: model_doc/openai-gpt` under the Gemma2 title here

Reply:
I have resolved it! Thank you :) @stevhliu